### PR TITLE
A few more Either FAQ details

### PR DESCRIPTION
**Pull request by travisbrown**
In response to @philwills's [comment](https://github.com/typelevel/cats/pull/1412#issuecomment-256167580) on #1412.

### CODE REVIEW

**Comment by travisbrown** [comment]
@philwills I just merged in your change—let me know if this looks okay to you and we can close #1429.

File: docs/src/main/tut/faq.md [hunk 1]
```diff
@@ -40,9 +40,12 @@ a bunch of useful combinators around it. In Scala 2.12.x `Either`
 `scala.util.Either` and fill in the gaps via
 [syntax enrichment](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/syntax/either.scala).
 
-This syntax can be imported via `cats.syntax.either._` or through `cats.implicits._`.
+<<<<<<< HEAD
```
Comment (peterneyens) [inline docs/src/main/tut/faq.md]: Missed something :wink:

**Comment by adelbertc** [comment]
👍

**Comment by peterneyens** [comment]
:+1:

**Comment by philwills** [comment]
:+1: looks perfect to me.

**Comment by non** [comment]
👍

**Pull request merged**